js/ui/pivot_grid/utils/calculate_scrollbar_width.js

```diff
@@ -0,0 +1,18 @@
+import domAdapter from '../../../core/dom_adapter';
+import callOnce from '../../../core/utils/call_once';
+import { getScrollbarWidth } from './get_scrollbar_width';
+
+const calculateScrollbarWidth = callOnce(function() {
+    const document = domAdapter.getDocument();
+
+    document.body.insertAdjacentHTML('beforeend',
+        '<div style=\'position: absolute; overflow: scroll; width: 100px; height: 100px; top: -9999;\'></div>');
+
+    const scrollbar = document.body.lastElementChild;
+    const scrollbarWidth = getScrollbarWidth(scrollbar);
```

Comment:
We don't change scrollbar size in our css and imho we can use a standalone scrollbar instead of creating '$dummyScrollable.dxScrollable(options).dxScrollable('instance');'

Reply:
The browser calculate scrollbar size for varios zoom values. So we need calculate its size for initial setting of the current page.